picard/ui/options/cdlookup.py

```diff
@@ -70,9 +70,12 @@ def load(self):
     def save(self):
         if AUTO_DETECT_DRIVES:
             config.setting["cd_lookup_device"] = self.ui.cd_lookup_device.currentText()
+            drives = self.drives
         else:
-            config.setting["cd_lookup_device"] = self.ui.cd_lookup_device.text()
-        self.tagger.window.update_cd_lookup_drives(self.drives)
+            selected_drive = self.ui.cd_lookup_device.text()
+            config.setting["cd_lookup_device"] = selected_drive
+            drives = [selected_drive]
+        self.tagger.window.update_cd_lookup_drives(drives)
```

Comment:
We can prolly reduce code redundancy here, but I'm not sure why we have `self.ui.cd_lookup_device.currentText()` vs `self.ui.cd_lookup_device.text()`:

```python
    def save(self):
        selected_drive = self.ui.cd_lookup_device.text()
        config.setting["cd_lookup_device"] = selected_drive
        if AUTO_DETECT_DRIVES:
            drives = self.drives
        else:
            drives = [selected_drive]
        self.tagger.window.update_cd_lookup_drives(drives)
```

Reply:
That's because in one case self.ui.cd_lookup_device is a QComboBox and in the other case a QLineEdit. We could add `_set_selected_drive()` / `_get_selected_drive()` methods to unify this a bit.

Or just do a 

```
def save(self):
        if AUTO_DETECT_DRIVES:
            selected_drive = self.ui.cd_lookup_device.currentText()
            drives = self.drives
        else:
            selected_drive = self.ui.cd_lookup_device.text()
            drives = [selected_drive]
        config.setting["cd_lookup_device"] = selected_drive
        self.tagger.window.update_cd_lookup_drives(drives)
```

But really we don't win much here

Reply:
> Or just do a
> 
> ```
> def save(self):
>         if AUTO_DETECT_DRIVES:
>             selected_drive = self.ui.cd_lookup_device.currentText()
>             drives = self.drives
>         else:
>             selected_drive = self.ui.cd_lookup_device.text()
>             drives = [selected_drive]
>         config.setting["cd_lookup_device"] = selected_drive
>         self.tagger.window.update_cd_lookup_drives(drives)
> ```
> 
> But really we don't win much here

Thanks for the explanation, I think this last version is clearer though.